### ISSUE TITLE
Status of the Project

### ISSUE THREAD

**Issue by ghost** [wp-bootstrap/wp-bootstrap-navwalker#151]
There is not a lot of activity here lately. Are you considering looking at the PRs and issues any time soon? What about conceding the maintenance to someone willing to take on the project?

Cheers.

**Comment by pattonwebz** [comment]
Hey bucaran,

The original creator of the navwalker (@twittem) is pretty busy these days. Some of the previous collaborators have been added to the repo so that they can help with PRs and issues however seems we've (me included) not been dropping by often enough or putting in enough time dealing with things.

I'll take a quick look through some issues and PRs now. 

I'll leave this issue open for the moment in case Ed or anyone else wants to comment.

**Comment by ghost** [comment]
Thanks!  

If you have the time, have a look at #129. I suggest there using the default [WP filter API](https://github.com/twittem/wp-bootstrap-navwalker/pull/129) instead of the current approach. The author claims my update misses some features from the original, but never points out which ones when asked.

### tl;dr

It is more robust to rely on Wordpress' filter/hook framework than to subclass Walker_Nav_Menu. This avoids code duplication and arguably increases readability. Updated class is used and behaves exactly like the original class.

**Comment by pattonwebz** [comment]
Hey again,

I created a separate dev branch in this repo where I'll merge your PRs and see what the results are. I don't really have much time spare over the next few days but as soon as I get a spare hour I'll merge your changes to the new dev branch and then test to make sure it works.

When Ed says that you missed some features of the navwalker I believe he's probably talking about the ability to add headers, dividers, disabled links etc. I haven't taken a close look at your PR so I'm not certain if it does support those items or not but I think that's what he means when he says it's missing features.

**Comment by ghost** [comment]
:arrow_right: #129

**Comment by auginator** [comment]
Just a quick note - the images are not showing up on the project page: https://github.com/twittem/wp-bootstrap-navwalker

**Comment by SideQuestDIY** [comment]
@auginator This is true. They were hosted by @github and they broke them, I don't actually have a copy of the images anymore so they will need to be updated.

**Comment by bhubbard** [comment]
Hey guys, 

So I'm working with @twittem to help manage the project, under a github org called wp-bootstrap. We updated to version 2.0.5 which had many improvements to meet WordPress Coding Standards.  Gonna close this issue as the references to the broken images were removed, we can add them back later if it is requested/needed.
